cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,108 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  const faker = require('faker');
 
+  const worker = {
+    name: faker.name.firstName(),
+    lastName: faker.name.lastName(),
+    email: faker.internet.email(),
+    age: faker.random.number({ min: 18, max: 65 }),
+    salary: faker.random.number({ min: 2000, max: 25000 }),
+    department: faker.commerce.department()
+  };
+
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should have pagination', () => {
+    cy.get('.-pagination').should('exist');
+  });
+
+  it('should provide the ability to select rows count', () => {
+    cy.get('[aria-label="rows per page"]').select('5');
+    cy.get('[aria-label="rows per page"]').select('10');
+    cy.get('[aria-label="rows per page"]').select('20');
+  });
+
+  it('should provide the ability to add new worker', () => {
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(worker.name);
+    cy.get('#lastName').type(worker.lastName);
+    cy.get('#userEmail').type(worker.email);
+    cy.get('#age').type(worker.age);
+    cy.get('#salary').type(worker.salary);
+    cy.get('#department').type(worker.department);
+
+    cy.get('#submit').click();
+    cy.get('.rt-table').should('contain', worker.name, worker.lastName);
+  });
+
+  it('should provide the ability to delete worker', () => {
+    cy.get('#delete-record-1 > svg > path').click();
+    cy.get('.rt-table').should('not.contain', 'Cierra, Vega');
+  });
+
+  it('should provide the ability to delete all workers', () => {
+    cy.get('#delete-record-1 > svg > path').click();
+    cy.get('#delete-record-2 > svg > path').click();
+    cy.get('#delete-record-3 > svg > path').click();
+    cy.on('window:alert', (str) => {
```

Comment:
this assertion doesn't work because its not an alert

Reply:
fixed